packages/doltgres/content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -226,7 +272,430 @@ WHERE name = 'origin';
++-------------+--------- +
+```
+
+### Example Query
+
+The following query shows the commits reachable from the current checked out head and created by user `jennifersp` since April, 2022:
+
+{% embed url="https://www.dolthub.com/repositories/dolthub/first-hour-db/embed/main?q=SELECT+*%0AFROM+dolt_log%0AWHERE+committer+%3D+%22jennifersp%22+and+date+%3E+%222022-04-01%22%0AORDER+BY+date%3B" %}
+
+## Database Diffs
+
+### `dolt.diff`
+
+The `dolt.diff` (also known as `dolt_diff`) system table shows which tables in the current database were changed in each commit reachable from the active branch's HEAD. When multiple tables are changed in a single commit, there is one row in the `dolt.diff` system table for each table, all with the same commit hash. Any staged or unstaged changes in the working set are included with the value `WORKING` for their `commit_hash`. After identifying the tables that changed in a commit, the `dolt_diff_$TABLENAME` system tables can be used to determine the data that changed in each table.
+
+### Schema
+
+The `DOLT.DIFF` system table has the following columns
+
+```sql
++---------------+----------+
+| field         | Type     |
++---------------+----------+
+| commit_hash   | text     |
+| table_name    | text     |
+| committer     | text     |
+| email         | text     |
+| date          | datetime |
+| message       | text     |
+| data_change   | boolean  |
+| schema_change | boolean  |
++---------------+----------+
+```
+
+### Query Details
+
+`dolt.diff` displays the changes from the current branch HEAD, including any working set changes. If a commit did not
+make any changes to tables _(e.g. an empty commit)_, it is not included in the `dolt.diff` results.
+
+### Example Query
+
+Taking the
+[`dolthub/first-hour-db`](https://www.dolthub.com/repositories/dolthub/first-hour-db)
+database from [DoltHub](https://www.dolthub.com/) as our
+example, the following query uses the `dolt.diff` system table to find all commits, and the tables they changed,
+from the month of April, 2022.
+
+{% embed url="https://www.dolthub.com/repositories/dolthub/first-hour-db/embed/main?q=SELECT+commit_hash%2C+table_name%2C+schema_change%0AFROM+++dolt_diff%0AWHERE++date+BETWEEN+%222022-04-01%22+AND+%222022-04-30%22%3B%0A" %}
+
+From these results, we can see there were four commits to this database in October, 2020. Commits
+`224helo` only changed the `dolt_schemas` table, commit `7jrvg1a` changed the `dolt.docs`
+table, and commit `5jpgb0f` made changes to two tables. We can also see which of these tables had changes made to their schemas vs just data.
+
+To dig deeper into these changes, we can query
+the `dolt_diff_$TABLENAME` system tables specific to each of the changed tables, like this:
+
+{% embed url="https://www.dolthub.com/repositories/dolthub/first-hour-db/embed/main?q=SELECT+count%28*%29+as+total_rows_changed%0AFROM+++dolt_diff_dolt_schemas%0AWHERE++to_commit%3D%27224helolb2bg6iqrf9b7befrflehqgnb%27%3B%0A" %}
```

Comment:
Not a huge deal, but using a user-table instead of dolt_schemas would be better here

Reply:
Same for other examples